### PR TITLE
Epoll support

### PR DESCRIPTION
**Pull request by oshai**

### CODE REVIEW

File: postgresql-async/build.gradle [hunk 1]
```diff
@@ -7,6 +7,7 @@ dependencies {
     compile "org.joda:joda-convert:$joda_convert_version"
     compile "io.netty:netty-transport:$netty_version"
     compile "io.netty:netty-handler:$netty_version"
+    compile "io.netty:netty-transport-native-epoll:$netty_version:linux-x86_64"
```
Comment (andy-yx-chen) [inline postgresql-async/build.gradle]: Can you add kqueue support as well? for macOs/freeBSD

Comment (oshai) [reply postgresql-async/build.gradle]: @andy-yx-chen I am not sure how usable that is (mac servers?). In addition, do you know if there is a way to check if it is supported (like Epoll.isAvailable())?

Comment (andy-yx-chen) [reply postgresql-async/build.gradle]: we can cover freeBSD servers, it's KQueue.isAvailable()

Comment (oshai) [reply postgresql-async/build.gradle]: @andy-yx-chen thanks, I added that as well

Comment (andy-yx-chen) [reply postgresql-async/build.gradle]: @oshai , thanks, looks pretty good

**Pull request merged**